frontend/src/components/Header.jsx

```diff
@@ -1,24 +1,34 @@
 import { Link } from "react-router-dom"
+import { useEffect, useState } from "react"
 
 import TextSearch from "./TextSearch"
 import gatsbyLogo from "../images/GATSBY.jpeg"
 import "../style/Header.css"
 
 export default function Header({ searchHandler }) {
+  const [currentPage, setCurrentPage] = useState()
+
+  useEffect(() => {
+    const page = window.location.href
+    const pageRegex = /(about)|(search)/g
+
+    setCurrentPage(page.match(pageRegex)[0])
+  }, [window.location.href])
```

Comment:
Getting a eslint error with this dependency error:
React Hook useEffect has an unnecessary dependency: 'window.location.href'. Either exclude it or remove the dependency array. Outer scope values like 'window.location.href' aren't valid dependencies because mutating them doesn't re-render the component.

Reply:
eslint is technically right, except that this useEffect is tracking the url to check which navigation button to highlight. So that does re-render when the location changes, because this component is part of our layout that persists on all pages. 

Removing this dependency breaks that functionality. We could track the params, but those won't apply if we are navigating out of '/search'